hadoop-ozone/ozonefs-shaded/pom.xml

```diff
@@ -120,38 +129,47 @@
                 <relocation>
                   <pattern>org</pattern>
                   <shadedPattern>${shaded.prefix}.org</shadedPattern>
-                  <includes>
-                    <include>org.yaml.**.*</include>
-                    <include>org.sqlite.**.*</include>
-                    <include>org.tukaani.**.*</include>
-                    <include>org.bouncycastle.**.*</include>
-                    <include>org.rocksdb.**.*</include>
-                    <include>org.apache.commons.cli.**.*</include>
-                    <include>org.apache.commons.compress.**.*</include>
-                    <include>org.apache.commons.codec.**.*</include>
-                    <include>org.apache.commons.beanutils.**.*</include>
-                    <include>org.apache.commons.collections.**.*</include>
-                    <include>org.apache.commons.digester.**.*</include>
-                    <include>org.apache.commons.io.**.*</include>
-                    <include>org.apache.commons.logging.**.*</include>
-                    <include>org.apache.commons.validator.**.*</include>
-                    <include>org.apache.commons.lang3.**.*</include>
-                    <include>org.sqlite.**.*</include>
-                    <include>org.apache.thrift.**.*</include>
-                  </includes>
+                  <excludes>
+                    <exclude>org.apache.commons.logging.**</exclude>
```

Comment:
```
org.apache.commons.logging.**
org.bouncycastle.**
```
These two are included to shade currently,  but excluded in the new change.  Is there any impact if not shaded?

Reply:
`org.apache.commons.logging.**` this can be removed from the exclude list, i will remove it.

Reply:
`org.bouncycastle.**` I am considering whether we should exclude this package from Ozone client release package.